docker/docker-compose-test.yaml

```diff
@@ -1,51 +1,63 @@
+# .env file. Careful that vscode black does not add spaces around '='
+#   COMPOSE_PROJECT_NAME='dlc'
+#   TEST_DATA_DIR=<local dir for from_top_tracking>
+#   GITHUB_USERNAME=datajoint
+#   INSTALL_OPTION=local-all, local-dlc, or git
+#   TEST_CMD="pytest" # pytest --dj-{verbose,teardown} False  # options
+#                     # to do nothing, set as "True"
 # export COMPOSE_DOCKER_CLI_BUILD=0 # some machines need for smooth --build
-# .env file: TEST_DATA_DIR=<local parent directory of workflow_dlc_data{1,2}>
-# docker-compose -f ./docker/docker-compose-test.yaml up --build
-# docker exec -it workflow-deeplabcut_workflow_1 /bin/bash
-# docker-compose -f ./docker/docker-compose-test.yaml down
+# docker-compose --env-file ./docker/.env -f ./docker/docker-compose-test.yaml up --build --force-recreate --detached
+# docker exec -it workflow-deeplabcut /bin/bash
+# docker-compose -f ./docker/docker-compose-test.yaml down --volumes
```

Comment:
```suggestion
# docker compose --env-file ./docker/.env -f ./docker/docker-compose-test.yaml up --build --force-recreate --detached
# docker exec -it workflow-deeplabcut /bin/bash
# docker compose -f ./docker/docker-compose-test.yaml down --volumes
```

Reply:
As part of this Docker refactor, let's switch from `docker-compose` to using the new `docker compose `.  Please make sure the flags here still work.

Reply:
Good catch. Yes, they still work. I switched in my own testing, but neglected to switch the instructions.